doc/admin.md

```diff
@@ -0,0 +1,193 @@
+installation.
+
+- 1.- Create a virtualenv 'deleteENV' invoking *virtualenv deleteENV*
+- 2.- Activate the virtualenv with *source deleteENV/bin/activate*
+- 3.- Install the requirements running *pip install -r requirements.txt
+   --allow-all-external*
+
+## Running FIWARE testbed deploy
+
+### Deploying one testbed
+To deploy just a testbed, it is required just execute one script. However some variables should be exported before.
+
+    export OS_AUTH_URL=<the authentication URL for the keystone in the Cloud>
+    export OS_USERNAME=<a user with an account in the Cloud>
+    export OS_TENANT_NAME=<a project name from the account in the Cloud>
+    export OS_PASSWORD=<the password for an account in the Cloud>
+    export OS_REGION_NAME=<the region name>
+    export OS_USER_DOMAIN_NAME=<OpenStack user domain name>
+    export OS_PROJECT_DOMAIN_NAME=<OpenStack project domain name>
+    export BOOKED_IP=<a booked IP in your Cloud infrastructure to deploy the VM>
+    export Region1=<the name for the region of the Cloud to be deployed>
+
+    fiwaretestbeddeploy/launch_vm.py
+
+The *launch_vm.py* ends in a few seconds, showing the floating IP. Then, it is
+ possible to connect to the ubuntu account of the server (using the
+SSH key at *~/.ssh/testbedskuld_key*).
+
+    ssh ubuntu@floating_ip
+
+Although the VM is active, the installation is still running
+inside the VM and needs a few minutes to complete. Usually the installation
+process lasts between 10 and 20 minutes. The job is finished after the file
+*config_vars* is copied into the */home/ubuntu* folder inside the virtual machine.
+
+When the installation is finished, the credential may be loaded with *. ~/config_vars*.
+The command *nova list* shows a testing VM that has been created during the installation
+inside the testbed (that is, it is a virtual machine running inside the testbed
+virtual machine). The floating IP 192.168.58.201 is associated to this
+VM (that is the second IP of the pool, the first was assigned to the router). It is
+possible to connect to the server following this steps:
+
+    eval $(ssh-agent -s)
+    ssh-add ~/.ssh/testkey
+    ip a add dev br-ex 192.168.58.1/24
+    ssh cirros@192.168.58.201
+
+Of course, if a new server is instantiated using the same network, there will
+be network visibility between both servers, according the rules of the
+security groups.
+
+     . ~/config_vars
+     NETID =
+     nova boot testvm2 --poll --flavor micro2 --image cirros --nic net-id=$NETID\
+      --key-name testkey --security-groups ssh
+     ssh -A cirros@192.168.58.201
+     $ ssh cirros@192.168.58.3
+
+The micro flavor provides 64MB of RAM, 1 VCPU and 1GB of disk. The micro2 flavor is the
+same, but with 0GB of disk (i.e. a minimal disk to boot the image is created
+but with barely free space)
+
+### Deploying three testbeds
+To deploy just a testsbed, it is required just execute one script. However some variables should be exported before.
```

Comment:
which one? You put the name of the script after the export variables. Something confuse for some readers.